geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/commands/ConnectCommand.java

```diff
@@ -168,26 +168,27 @@ public ResultModel connect(
       return result;
     }
 
-    // since 1.14, only allow gfsh to connect to cluster that's older than 1.10
     String remoteVersion = null;
-    String gfshVersion = gfsh.getVersion();
+    String remoteGeodeSerializationVersion = null;
     try {
       remoteVersion = invoker.getRemoteVersion();
-      int minorVersion = Integer.parseInt(versionComponent(remoteVersion, VERSION_MINOR));
-      if (versionComponent(remoteVersion, VERSION_MAJOR).equals("1") && minorVersion >= 10 ||
-          versionComponent(remoteVersion, VERSION_MAJOR).equals("9") && minorVersion >= 9) {
-        InfoResultModel versionInfo = result.addInfo("versionInfo");
-        versionInfo.addLine("You are connected to a cluster of version: " + remoteVersion);
-        return result;
-      }
+      remoteGeodeSerializationVersion = getRemoteSerializationVersion(invoker);
```

Comment:
This is fine, but if you just call the invoker method directly, and user gfsh.logToFile in the catch block, things should be fine too. The difference is that user won't see this "failed to get the remote version" message when they connect to pre-1.5 cluster, but they would still see "you can't use a xxx version gfsh to connect to this cluster"